plugins/repository-hdfs/src/test/java/org/elasticsearch/plugin/hadoop/hdfs/UtilsTests.java

```diff
@@ -0,0 +1,11 @@
+package org.elasticsearch.plugin.hadoop.hdfs;
+
+import org.elasticsearch.test.ESTestCase;
+
+public class UtilsTests extends ESTestCase {
```

Comment:
Added an actual unit test instead of the dummy one so even if the integration tests are not running, there are some tests in place and the build can succeed.